robot/demos/SkidSteerServos/SkidSteeringDC.ino

```diff
@@ -0,0 +1,93 @@
+#include <ArduinoBlue.h>
+#include <SoftwareSerial.h>
+#include <Servo.h>
+
+int prevThrottle = 49;
+int prevSteering = 49;
+int throttle, steering;
+// Initialize Left & Right Velocity variables to Rest
+int VR = 127;
+int VL = 127;
+SoftwareSerial bluetooth(7, 8);
+
+ArduinoBlue phone(bluetooth);
+
+void setup() {
+  // Attach Servos to pins:
+  // Right side servos
+  DC_PIN_1 = 9;
+  DC_PIN_2 = 10;
+  DC_PIN_3 = 11;
+  
+  //Left side Servos
+  DC_PIN_4 = 3;
+  DC_PIN_5 = 5;
+  DC_PIN_6 = 6;
+
+  // initialize serial communications at 9600 bps:
+  Serial.begin(9600); 
+  bluetooth.begin(9600);
+
+  //activate pull-up resistor on the push-button pin
+  pinMode(buttonPin, INPUT_PULLUP); 
+  
+  Serial.println("setup complete");
+}
+
+float deg = 0;
```

Comment:
I could have mentioned it for the other file as well but I find it a little weird that you make new variable definitions between setup() and loop(). I'd put everything at the top.

Reply:
Lol yea I see what you mean, I think its just a bad habit. Fixed!